### PR TITLE
Enable VIA - ALF X1.1

### PR DESCRIPTION
**Pull request by drowe**
## Description

Enabling VIA and some minor product name updates for clarity / accuracy.

## Types of Changes

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [ ] Documentation

## Issues Fixed or Closed by This PR

N/A

## Checklist

- [x] My code follows the code style of this project: [**C**](https://docs.qmk.fm/#/coding_conventions_c), [**Python**](https://docs.qmk.fm/#/coding_conventions_python)
- [x] I have read the [**PR Checklist** document](https://docs.qmk.fm/#/pr_checklist) and have made the appropriate changes.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/alf/x11/info.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
 {
-  "keyboard_name": "x11",
-  "manufacturer": "Alf",
+  "keyboard_name": "ALF X1.1",
```
Comment (fauxpark) [inline keyboards/alf/x11/info.json:2]:
```suggestion
  "keyboard_name": "X1.1",
```

File: keyboards/alf/x11/readme.md [hunk 2]
```diff
@@ -10,4 +10,8 @@ Make example for this keyboard (after setting up your build environment):
 
     make alf/x11:default
 
+If you'd like to build a VIA-supported firmware, use this example:
+    
+    make alf/x11:via
+
```
Comment (fauxpark) [inline keyboards/alf/x11/readme.md:16]:
```suggestion
```

File: keyboards/alf/x11/keymaps/via/readme.md [hunk 3]
```diff
@@ -0,0 +1 @@
+The default VIA layout for ALF X1.1
```
Comment (fauxpark) [inline keyboards/alf/x11/keymaps/via/readme.md:1]: This file can be deleted.

File: keyboards/alf/x11/keymaps/default/readme.md [hunk 4]
```diff
@@ -1 +1 @@
-# The default keymap for x11
+# The default keymap for ALF X1.1
```
Comment (fauxpark) [inline keyboards/alf/x11/keymaps/default/readme.md:1]: This file can be deleted.

**Review by fauxpark — CHANGES_REQUESTED** [review]

**Review by fauxpark — APPROVED** [review]

**Review by drashna — APPROVED** [review]

**Pull request merged**